### ISSUE TITLE
Update style of existing Highlights

### ISSUE THREAD

**Issue by k88hudson** [mozilla/activity-stream#1190]
This is for Highlights:
![highlights](https://cloud.githubusercontent.com/assets/1455535/18097627/9a27e424-6eac-11e6-811f-a50dbc149795.jpg)

Invision is here: https://mozilla.invisionapp.com/share/TN84I7MB2#/screens/177705068

NOTE: needs update, waiting on invision

**Comment by jaredkerim** [comment]
Assigning to @bryanbell for quick design review before the green light for implementation.

**Comment by k88hudson** [comment]
After a meeting today, we decided to update the requirements here. @bryanbell will have a revised invision shortly!

**Comment by k88hudson** [comment]
![image](https://cloud.githubusercontent.com/assets/1455535/18446431/4c29fab6-78f0-11e6-80b3-08d226cd1317.png)

**Comment by bryanbell** [comment]
!(https://mozilla.invisionapp.com/static-signed/live-embed/22240415/177705068/5/latest/hwRBAB4lEfnJi8lDy1lEeLQizReEa9mTBfDlEgvN9xnKPjy9WQiTznMSgQszrrByduOlEeYii7RGCPHtchm1yjSxdwlE/NewTab---More-Highlights-2x.png)

!(https://mozilla.invisionapp.com/static-signed/live-embed/22240415/189946144/1/latest/56lEZ1MU5gy1hsnr29EdlELHDY2O9eNw2RnTtK97kIR258KlPFZtcQtN3q6uSGlEl36uu2gXsWbOKHmUvd9u8j7hAlE/Measurements---Highlight-2x.png)

**Comment by k88hudson** [comment]
Let's do the second version (rows of `Top Highlight`s only, no `Bottom Highlight`s) as a separate variant of the `weightedHighlights` experiment. This will be possible once #1312 lands. Feel free to split into a separate ticket if it makes more sense
